packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -5205,7 +5185,11 @@ export class WalletService implements IWalletService {
 
   moralisGetTokenAllowance(req): Promise<any> {
     return new Promise((resolve, reject) => {
-      let keys, headers;
+      const headers = {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        'X-Api-Key': config.moralis.apiKey,
+      };
```

Comment:
can you move this object back down to where it was but just declare it as a const down there? That way `config.moralis.apiKey` isn't evaluated before `!config.moralis` is checked (the next line).